[2025-05-07T03:10:21.805+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: lgbtq_model_pipeline.retrain manual__2025-05-07T03:07:53.793360+00:00 [queued]>
[2025-05-07T03:10:21.809+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: lgbtq_model_pipeline.retrain manual__2025-05-07T03:07:53.793360+00:00 [queued]>
[2025-05-07T03:10:21.809+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-05-07T03:10:21.814+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): retrain> on 2025-05-07 03:07:53.793360+00:00
[2025-05-07T03:10:21.820+0000] {standard_task_runner.py:57} INFO - Started process 1717 to run task
[2025-05-07T03:10:21.822+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'lgbtq_model_pipeline', 'retrain', 'manual__2025-05-07T03:07:53.793360+00:00', '--job-id', '23', '--raw', '--subdir', 'DAGS_FOLDER/lgbtq_model_pipeline.py', '--cfg-path', '/tmp/tmp6hoyq60m']
[2025-05-07T03:10:21.823+0000] {standard_task_runner.py:85} INFO - Job 23: Subtask retrain
[2025-05-07T03:10:21.848+0000] {task_command.py:416} INFO - Running <TaskInstance: lgbtq_model_pipeline.retrain manual__2025-05-07T03:07:53.793360+00:00 [running]> on host 6fdcfcafebfc
[2025-05-07T03:10:21.881+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='lgbtq_model_pipeline' AIRFLOW_CTX_TASK_ID='retrain' AIRFLOW_CTX_EXECUTION_DATE='2025-05-07T03:07:53.793360+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-07T03:07:53.793360+00:00'
[2025-05-07T03:10:21.882+0000] {logging_mixin.py:154} INFO - üì¶ Combining data...
[2025-05-07T03:10:22.067+0000] {logging_mixin.py:154} INFO - ‚ö†Ô∏è Skipping duplicate article_246
[2025-05-07T03:10:22.068+0000] {logging_mixin.py:154} INFO - ‚ö†Ô∏è Skipping duplicate article_247
[2025-05-07T03:10:22.068+0000] {logging_mixin.py:154} INFO - ‚ö†Ô∏è Skipping duplicate article_245
[2025-05-07T03:10:22.069+0000] {logging_mixin.py:154} INFO - ‚ö†Ô∏è Skipping duplicate article_249
[2025-05-07T03:10:22.070+0000] {logging_mixin.py:154} INFO - ‚ö†Ô∏è Skipping duplicate article_248
[2025-05-07T03:10:22.070+0000] {logging_mixin.py:154} INFO - ‚ö†Ô∏è Skipping duplicate article_244
[2025-05-07T03:10:22.070+0000] {logging_mixin.py:154} INFO - üìö Loading labeled data...
[2025-05-07T03:10:22.155+0000] {logging_mixin.py:154} INFO - üß† Preparing triplet training data...
[2025-05-07T03:10:22.156+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/tasks/retrain.py", line 62, in train_triplet_model
    triplet_dataset = get_triplet_dataset(texts, labels, model_path=HF_MODEL_PATH)
TypeError: get_triplet_dataset() got an unexpected keyword argument 'model_path'
[2025-05-07T03:10:22.161+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=lgbtq_model_pipeline, task_id=retrain, execution_date=20250507T030753, start_date=20250507T031021, end_date=20250507T031022
[2025-05-07T03:10:22.166+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 23 for task retrain (get_triplet_dataset() got an unexpected keyword argument 'model_path'; 1717)
[2025-05-07T03:10:22.209+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2025-05-07T03:10:22.224+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
